tests/e2e/vmgroups_test.go

```diff
@@ -674,7 +674,7 @@ func (vg *VmGroupTest) TestVmgroupRemoveWithRemoveVol(c *C) {
 // 	}
 
 // 	log.Printf("Removing test vmgroup %s", vgTestVMgroup1)
-// 	out, err = adminutils.DeleteVMgroup(vg.config.EsxHost, vgTestVMgroup1, false)
+// 	out, err = adminutils.DeleteVMgroup(vg.config.EsxHost, vgTestVMgroup1, true)
```

Comment:
shouldn't this test be removed? with reference `TODO: Need to enable or remove the following tests after we have conclusion on issue #1469` @line#632

Reply:
+1